### PR TITLE
Revamp the Database list page

### PR DESCRIPTION
**Pull request by MauricioFauth**
A small revamp of the _Server -> Databases_ page.

### CODE REVIEW

**Comment by williamdes** [comment]
@sudo-robot deploy

File: themes/bootstrap/img/database-minus.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg"><path d="m16 2c-6.6 0-12 2.2-12 5v4c0 2.8 5.4 5 12 5s12-2.2 12-5v-4c0-2.8-5.4-5-12-5zm0 2c5 0 9 1.6 9 3.5s-4 3.5-9 3.5-9-1.6-9-3.5 4-3.5 9-3.5zm-12 9v5c0 2.8 5.4 5 12 5h.2c.5-2.1 1.8-3.9 3.5-5.2-1.2.2-2.4.2-3.7.2-6.6 0-12-2.2-12-5zm24 0c0 1.1-.9 2.1-2.5 3 .8.1 1.7.2 2.5.5zm-24 7v5c0 2.8 5.4 5 12 5 .5 0 1 0 1.5-.1-1-1.5-1.5-3.1-1.5-4.9-6.6 0-12-2.2-12-5z"/><path d="m25 18c-3.9 0-7 3.1-7 7s3.1 7 7 7 7-3.1 7-7-3.1-7-7-7zm5 6v2h-10v-2z"/></svg>
```
Comment (williamdes) [inline themes/bootstrap/img/database-minus.svg:1]: could you add this to the LICENSE documentation vendors.rst file ?

**Review by williamdes — APPROVED** [review]
:100:

File: themes/bootstrap/img/database-minus.svg (hunk 1 above)
Comment (mauriciofauth) [reply themes/bootstrap/img/database-minus.svg:1]: It is this icon: https://github.com/phpmyadmin/data/blob/master/resources/svg/icons/database-minus.svg

Comment (williamdes) [reply themes/bootstrap/img/database-minus.svg:1]: Oh okay, perfect !
Maybe you still can add a section and reference the PR that added them
No idea about the license
Do what will seem best and merge when you want ;)

**Pull request merged**